src/main/scala/uk/gov/ons/bi/dataload/ubrn/LinksPreprocessor.scala

```diff
@@ -0,0 +1,91 @@
+package uk.gov.ons.bi.dataload.ubrn
+
+import com.google.inject.Singleton
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.{DataFrame, SQLContext}
+import org.apache.spark.sql.functions._
+import uk.gov.ons.bi.dataload.reader.LinkJsonReader
+import uk.gov.ons.bi.dataload.utils.AppConfig
+
+import scala.util.{Success, Try}
+
+/**
+  * Created by websc on 03/03/2017.
+  */
+
+@Singleton
+class LinksPreprocessor(sc: SparkContext) {
+
+  val defaultBaseUbrn = 1L
```

Comment:
True.  We can change this later, if necessary.  But there is no guarantee we can keep within 12-digit range (see below) as we are using Spark's Monotonically Increasing ID function to generate a 64-bit Long value, which is partly based on the physical node as well as a simple counter.  If we have to process data on multiple nodes, we get different ranges of numbers (Poss over 12 digits) anyway.